mvlearn/preprocessing/preprocess.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Authors: Alexander Chang, Gavin Mischler
+
+import numpy as np
+from sklearn import model_selection
+from itertools import chain
+from ..utils.utils import check_Xs_y, check_Xs
+
+
+def train_test_split(Xs, y=None, test_size=None, train_size=None,
+                     random_state=None, shuffle=True, stratify=None):
```

Comment:
why don't you stick to the sklearn public API?

https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

in sklearn you can also split sample_weights, groups etc...

Reply:
ok, i'll make sure it can deal with arbitrary numbers of multiview or single-view arrays